docs/api-reference/io-columns-binary-classification.md

```diff
@@ -0,0 +1,8 @@
+### Input and Output Columns
```

Comment:
Looks the information below is not related to `Input`? Maybe you want to add the two types of inputs for binary classifiers --- single float column and multiple float columns (FFM).

Reply:
the input information is the first line above the table about label. Because it doesn't have a fixed name (unlike output columns) it's just written in text.

FFM is a special binary classifier. For that we can add the multiple float input.

---
In reply to: [275443192](https://github.com/dotnet/machinelearning/pull/3310#discussion_r275443192) [](ancestors = 275443192)